src/utils.ts

```diff
@@ -320,6 +320,21 @@ export function linkedQ(
   } else return fromTo;
 }
 
+/**
+ * Check if the link `from` → `to` is resolved or not.
+ * @param  {App} app
+ * @param  {string} to
+ * @param  {string} from
+ * @returns boolean
+ */
+export function unresolvedQ(app: App, to: string, from: string): boolean {
```

Comment:
I think we should be using more self-explanatory variable/function names for this public library. I mean in most cases you can guess what it means from the tsdoc, but here I still don't know what `Q` ist for.

Reply:
Fair, it's mostly a convention I use in my own code. `Q` is for Question. So `unresolvedQ` tells you if these links are unresolved. `linkedQ` tells you if these notes are linked.
I'm open to using a different naming convention though, what do you think?

Reply:
Not really sure, how about something like `notesLinked()`, or `linkResolved()`, of course that is the opposite of what this does

Reply:
Obsidian uses `is` for boolean questions. So maybe `isLinked` or `isUnresolved` (which I easily turn into `isResolved`)

Reply:
Even better!